src/Sylius/Bundle/ApiBundle/Resources/config/services/providers.xml

```diff
@@ -15,12 +15,18 @@
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
 >
+    <parameters>
+        <parameter key="sylius.api_path_prefixes" type="collection">
+            <parameter type="constant">Sylius\Bundle\ApiBundle\Provider\PathPrefixes::ADMIN_PREFIX</parameter>
+            <parameter type="constant">Sylius\Bundle\ApiBundle\Provider\PathPrefixes::SHOP_PREFIX</parameter>
+        </parameter>
```

Comment:
Would be nice to have this handled via configuration.
Also, note it in the upgrade file.

Reply:
I've added a note to the UPGRADE file 👌🏻 

About handling these path prefixes via configuration, I would leave the refactor for a separate PR, especially, that I'm not sure if it makes sense here 🤔